README.md

```diff
@@ -27,10 +27,17 @@ conda activate movement-env
 
 ## Overview
 
-Pose estimation tools, such as [DeepLabCut](https://www.mackenziemathislab.org/deeplabcut) and [SLEAP](https://sleap.ai/) are now commonplace when processing video data of animal behaviour. There is not yet a standardised, easy-to-use way to process the pose tracks produced from these software packages.
-
-movement aims to provide a consistent modular interface to analyse pose tracks, allowing steps such as data cleaning, visualisation and motion quantification.
-We aim to support a range of pose estimation packages, along with 2D or 3D tracking of single or multiple individuals.
+Machine learning-based tools such as
+[DeepLabCut](https://www.mackenziemathislab.org/deeplabcut) and
+[SLEAP](https://sleap.ai/) have become commonplace for tracking the
+movements of animals and their body parts in videos.
+However, there is still a need for a standardized, easy-to-use method
+to process the tracks generated by these tools.
+
+`movement` aims to provide a consistent, modular interface for analyzing
+motion tracks, enabling steps such as data cleaning, visualization,
+and motion quantification. We aim to support a range of animal tracking
+frameworks, as well as 2D or 3D tracking of single or multiple individuals.
```

Comment:
It's not clear to me how "a range of animal tracking frameworks" differs from "2D or 3D tracking of single of multiple individuals". 

How about "We aim to support all popular animal tracking frameworks and common file formats". "Aim" being the key word!

Reply:
Ambitious, but I like it.